app/forms/RemoveAreYouSureFormProvider.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import javax.inject.Inject
+
+import forms.mappings.Mappings
+import play.api.data.Form
+
+class RemoveAreYouSureFormProvider @Inject() extends Mappings {
```

Comment:
why it needs `@Inject()` - 
I see we have used it other places as well, but can't figure out yet as why?

Reply:
to use it in controller injection - got it :)

Reply:
Aye i believe this comes in in the scaffold and is easily overlooked - might be worth treating as a bit of tech debt to clean up our files for uneeded tags